test/embed.test.js

```diff
@@ -161,28 +161,28 @@ describe('Embed Tests', () => {
 
   it('Sanitizes Malicious URLs', async () => {
     // eslint-disable-next-line no-script-url
-    const { headers, body } = await embed('javascript:alert(1)');
+    const { headers, body } = await embed('javascript:alert(1)', { kind: 'embed-' });
     assert.equal(headers['Content-Type'], 'text/html');
     assertContains(body, ['about:blank']);
   });
 
   it('Fails Gracefully', async () => {
-    const { body } = await embed('https://unsplash.com/photos/0lD9SSMC6jo', { UNSPLASH_AUTH: 'superFake' });
+    const { body } = await embed('https://unsplash.com/photos/0lD9SSMC6jo', { UNSPLASH_AUTH: 'superFake', kind: 'embed-unsplash' });
     assertContains(body, ['<a href="https://unsplash.com/photos/0lD9SSMC6jo">']);
   });
 
   it('Supports Lottifiles', async () => {
-    const { body } = await embed('https://lottiefiles.com/17003-control-animated-volume-1');
+    const { body } = await embed('https://lottiefiles.com/17003-control-animated-volume-1', { kind: 'embed-lottiefiles' });
     assertContains(body, ['<script src="https://unpkg.com/@lottiefiles/lottie-player@latest/dist/lottie-player.js']);
   });
 
   it('Lottifiles Fails Gracefully', async () => {
-    const { body } = await embed('https://lottiefiles.com/this-will-definitely-fail-helix-141343151');
-    assertContains(body, ['<div class="embed  embed-has-url embed-has-title embed-has-image embed-has-description"']);
+    const { body } = await embed('https://lottiefiles.com/this-will-definitely-fail-helix-141343151', { kind: 'embed-lottiefiles' });
+    assertContains(body, ['<div class="embed  embed-has-url embed-lottiefiles embed-has-title embed-has-image embed-has-description"']);
   });
 
   it('Supports Spotify', async () => {
-    const { body } = await embed('https://open.spotify.com/playlist/37i9dQZF1DWYWddJiPzbvb');
+    const { body } = await embed('https://open.spotify.com/playlist/37i9dQZF1DWYWddJiPzbvb', { kind: 'embed-spotify-open' });
```

Comment:
Hmm, this looks like a special case where we might consider omitting the 3rd level domain: if Spotify embeds are always served from `open.spotify.com`, we are not adding any additional value by adding `embed-spotify-open` to the class list, especially if there is already a `embed-spotify` class. 

Should we drop `open` in case of `spotify`?

Reply:
Yeah that's a great point; man how do you guys catch that kind of stuff? teach meeee, fixing this now.

Reply:
😄 I just look at it from the perspective of a web developer writing CSS for these embeds: anything that could feel weird or unnecessary to them should be avoided.

Reply:
Right; that's a good point. If I have to add another domain to the filter list I'll take @tripodsan suggestion and extend the domain list :D